### ISSUE TITLE
ConfigureRemotingForAnsible.ps1 fails to run on Windows 10 Powershell 5.

### ISSUE THREAD

**Issue by ghost** [ansible/ansible#22674]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
ConfigureRemotingForAnsible.ps1

##### ANSIBLE VERSION
N/A

I downloaded the script from link on the Ansible Windows Support web page (http://docs.ansible.com/ansible/intro_windows.html)

https://github.com/ansible/ansible/blob/devel/examples/scripts/ConfigureRemotingForAnsible.ps1

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
fresh install of Windows 10 Pro
powershell version:
Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      14393  953

##### SUMMARY
Attempted to run ConfigureRemotingForAnsible.ps1 that I downloaded from the Ansible Windows Support wed page https://github.com/ansible/ansible/blob/devel/examples/scripts/ConfigureRemotingForAnsible.ps1

NOTE: I tried the version from stable-2.3 branch and it worked fine.

##### STEPS TO REPRODUCE
Download the powershell script from https://github.com/ansible/ansible/blob/devel/examples/scripts/ConfigureRemotingForAnsible.ps1

run with Powershell in Administrator mode

##### EXPECTED RESULTS
script to run and enable ansible/winrm on this box

##### ACTUAL RESULTS
>powershell -File ConfigureRemotingForAnsible.ps1

Security warning
Run only scripts that you trust. While scripts from the internet can be useful, this script can potentially harm your
computer. If you trust this script, use the Unblock-File cmdlet to allow the script to run without this warning
message. Do you want to run ConfigureRemotingForAnsible.ps1?
[D] Do not run  [R] Run once  [S] Suspend  [?] Help (default is "D"): r
At ConfigureRemotingForAnsible.ps1:118 char:13
+ </a>        <a href="/explore" class="js-selected-navigation-item nav ...
+             ~
The '<' operator is reserved for future use.
At ConfigureRemotingForAnsible.ps1:120 char:13
+ </a>        <a href="/pricing" class="js-selected-navigation-item nav ...
+             ~
The '<' operator is reserved for future use.
At ConfigureRemotingForAnsible.ps1:122 char:11
+ </a>      </nav>
+           ~
The '<' operator is reserved for future use.
At ConfigureRemotingForAnsible.ps1:3323 char:225
+ ... g:0;display:inline"><input name="utf8" type="hidden" value="&#x2713;" ...
+                                                                 ~
The ampersand (&) character is not allowed. The & operator is reserved for future use; wrap an ampersand in double
quotation marks ("&") to pass it as part of a string.
At ConfigureRemotingForAnsible.ps1:3356 char:11
+       <li>&copy; 2017 <span title="0.08151s from github-fe163-cp1-prd ...
+           ~
The ampersand (&) character is not allowed. The & operator is reserved for future use; wrap an ampersand in double
quotation marks ("&") to pass it as part of a string.
At ConfigureRemotingForAnsible.ps1:3356 char:23
+       <li>&copy; 2017 <span title="0.08151s from github-fe163-cp1-prd ...
+                       ~
The '<' operator is reserved for future use.
At ConfigureRemotingForAnsible.ps1:3356 char:29
+ ...  2017 <span title="0.08151s from github-fe163-cp1-prd.iad.github.net" ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Unexpected token 'title="0.08151s from github-fe163-cp1-prd.iad.github.net">GitHub</span>' in expression or statement.
At ConfigureRemotingForAnsible.ps1:3356 char:100
+ ... 151s from github-fe163-cp1-prd.iad.github.net">GitHub</span>, Inc.</l ...
+                                                                 ~
Missing argument in parameter list.
At ConfigureRemotingForAnsible.ps1:3375 char:12
+     You can't perform that action at this time.
+            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The string is missing the terminator: '.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : RedirectionNotSupported

**Comment by jhawkesworth** [comment]
Hmm, looks like you have downloaded the github html page that shows the script, rather than the raw powershell script.

Click on 'Raw' on this page https://github.com/ansible/ansible/blob/devel/examples/scripts/ConfigureRemotingForAnsible.ps1 to get to the actual script contents

You can download directly from this url (which is what Raw points at)
https://raw.githubusercontent.com/ansible/ansible/devel/examples/scripts/ConfigureRemotingForAnsible.ps1
or use a tinyurl I set up for it

http://tinyurl.com/qjb8nbk

I'm going to go ahead and close this issue but please feel free to suggest a wording change in a new issue or create a documentation Pull Request to improve the documentation regarding this.
